package/MDAnalysis/core/topologyattrs.py

```diff
@@ -2296,14 +2309,22 @@ def set_atoms(self, ag):
         return NotImplementedError("Cannot set bond information")
 
     def get_atoms(self, ag):
+        """
+        Get subset for atoms.
```

Comment:
It is good to add a docstring, but I am not sure this one helps much: what subset of the atoms?

Reply:
I hope this new one is more helpful, although I'm not certain users will ever really see it